minecraft_servers/astrogames/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "server_name": "astrogames",
+  "nice_name": "AstroGames",
+  "direct_ip": "astrogames.net",
+  "server_wildcards": [
+    "%.AstroGames.net"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://astrogames.net",
```

Comment:
Website not loading...
<img width="302" alt="Bildschirmfoto 2023-05-10 um 21 10 53" src="https://github.com/LabyMod/server-media/assets/45363287/27b7896a-4554-4074-bd93-137ca62ea820">